subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/execution/CharSequenceExtensions.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.gradle.kotlin.dsl.provider
+package org.gradle.kotlin.dsl.execution
```

Comment:
API breaking change?

Reply:
These functions are not part of the API.